glpk/bld.bat

```diff
@@ -0,0 +1,14 @@
+cd w%ARCH%
+copy config_VC config.h
+
+nmake /f Makefile_VC
+
+mkdir %PREFIX%\bin

```

Comment:
Please add the testing, if possible.  That would be very nice to have.

On Mon, Feb 15, 2016 at 8:04 AM Juan Luis Cano Rodríguez <
notifications@github.com> wrote:

> In glpk/bld.bat
> <https://github.com/conda/conda-recipes/pull/481#discussion_r52901609>:
>
> > @@ -0,0 +1,14 @@
> > +cd w%ARCH%
> > +copy config_VC config.h
> > +
> > +nmake /f Makefile_VC
> > +
> > +mkdir %PREFIX%\bin
>
> Besides @snorfalorpagus <https://github.com/snorfalorpagus> published his
> own glpk recipe five days ago and includes some testing, so perhaps he
> wants to weigh in. Otherwise I'm just going to edit bld.bat as little as
> possible.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/conda/conda-recipes/pull/481/files#r52901609>.
>